redbot/cogs/modlog/modlog.py

```diff
@@ -112,7 +112,13 @@ async def case(self, ctx: commands.Context, number: int):
             if await ctx.embed_requested():
                 await ctx.send(embed=await case.message_content(embed=True))
             else:
-                await ctx.send(await case.message_content(embed=False))
+                message = _("{case}\n**Timestamp**: {timestamp}").format(
```

Comment:
The colon is bolder for other fields, so it should be bolded here too for consistency.
```suggestion
                message = _("{case}\n**Timestamp:** {timestamp}").format(
```